### PR TITLE
UINV-123 Add Export to accounting to Adjustments

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UINV-123

#### Screenshot
image
image

#### TODOS and Open Questions
- Could be merged after [MODINVOSTO-58](https://issues.folio.org/browse/MODINVOSTO-58) [PR](https://github.com/folio-org/acq-models/pull/237)

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js [hunk 1]
```diff
@@ -36,47 +35,43 @@ class SettingsAdjustmentsEditor extends Component {
...
+
     return (
       <Layer
         contentLabel="Adjustments editor"
         isOpen
       >
         <form
           id="settings-adjustments-form"
-          onSubmit={handleSubmit}
+          style={{ height: '100vh' }}
```
Comment (aliaksei-chumakou) [inline src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js:66]: actually, it's a hack. IMHO, Layer should be removed here and this hack might not be required

Comment (AndreiShumski2) [reply src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js:66]: it looks like this without `<Layer>`:
image

Comment (aliaksei-chumakou) [reply src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js:66]: k, leave as is

File: test/bigtest/tests/settings/setting-create-adjustment-test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import setupApplication from '../../helpers/setup-application';
+import SettingsAdjustmentFormInteractor from '../../interactors/SettingsAdjustmentFormInteractor';
+
+describe('Create new adjustment', function () {
+  setupApplication();
+  const setting = new SettingsAdjustmentFormInteractor();
+
+  beforeEach(function () {
+    this.visit('/settings/invoice/adjustments/create');
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/settings/setting-create-adjustment-test.js:12]: no await

File: test/bigtest/tests/settings/setting-create-adjustment-test.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  describe('Add data and save adjustment', () => {
+    beforeEach(async function () {
+      await setting.description.fill('test value');
+      await setting.formFooter.saveButton.click();
+    });
+
+    it('closes form', () => {
+      expect(setting.isPresent).to.be.false;
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/settings/setting-create-adjustment-test.js:26]: potentially this test might fail a lot by timeout issue. IMHO, the right pattern is to await that destination is loaded and available

**Pull request merged**